src/src/rx-serial/SerialSUMD.h

```diff
@@ -0,0 +1,27 @@
+#include "SerialIO.h"
+
+#define SUMD_HEADER_SIZE		3														// 3 Bytes header
+#define SUMD_DATA_SIZE_16CH		(16*2)													// 2 Bytes per channel
+#define SUMD_CRC_SIZE			2														// 16 bit CRC
+#define SUMD_FRAME_16CH_LEN		(SUMD_HEADER_SIZE+SUMD_DATA_SIZE_16CH+SUMD_CRC_SIZE)	
+
+#define CRC_POLYNOME 0x1021
```

Comment:
Move these into the CPP file as they are implementation details.

Reply:
done